assets/src/features/home-page/components/HomePageVideoTile.tsx

```diff
@@ -44,7 +44,7 @@ const HomePageVideoTile: React.FC<HomePageVideoTileProps> = ({ displayName }) =>
       disableGroupHover
       layers={
         <>
-          {!cameraAutostart.status ? <InitialsImage initials={initials} /> : null}
+          {!(cameraAutostart.status && localCamera.isEnabled) ? <InitialsImage initials={initials} /> : null}
```

Comment:
I think it's more readable when we have to negated terms joined with alternative than negated conjunction.

```suggestion
          {!cameraAutostart.status || !localCamera.isEnabled) ? <InitialsImage initials={initials} /> : null}
```

`If camera should not autostart or local camera is not enabled then show image`